test/WebSites/ModelBindingWebSite/Controllers/TryUpdateModelController.cs

```diff
@@ -98,6 +98,16 @@ public class TryUpdateModelController : Controller
             return user;
         }
 
+        public async Task<Employee> GetEmployeeAsync_TypeCastedAsPerson()
+        {
+            Person person = new Employee();
+            await TryUpdateModelAsync(
```

Comment:
nit:
1. Although this is perfectly correct, it might be clearer if it used the explicit `TryUpdateModelAsync<Person>` 
2. There is no cast, how about `GetEmployeeAsync_BindToBaseDeclaredType` ?